File: core/src/ca/kanoa/zombieworld/utilities/Size.java
package ca.kanoa.zombieworld.utilities;

import com.badlogic.gdx.Gdx;

public class Size {

    public static float getWidthCenti(float centimeters) {
        return Gdx.graphics.getPpcX() * centimeters;
    }

    public static float getHeightCenti(float centimeters) {
        return Gdx.graphics.getPpcY() * centimeters;
    }

    public static float convertY(float y) {
        return Gdx.graphics.getHeight() - y;
    }

    public static float getScreenWidth() {
        return Gdx.graphics.getWidth();
    }

    public static float getScreenHeight() {
        return Gdx.graphics.getHeight();
    }

}

File: core/src/ca/kanoa/zombieworld/utilities/Size.kt
package ca.kanoa.zombieworld.utilities

import com.badlogic.gdx.Gdx

object Size {

    fun getWidthCenti(centimeters: Float): Float = Gdx.graphics.ppcX * centimeters

    fun getHeightCenti(centimeters: Float): Float = Gdx.graphics.ppcY * centimeters

    fun convertY(y: Float): Float = Gdx.graphics.height - y

    val screenWidth: Float
        get() = Gdx.graphics.width.toFloat()

    val screenHeight: Float
        get() = Gdx.graphics.height.toFloat()

}
